pkg/apis/paddlepaddle/v1/trainingjob.go

```diff
@@ -0,0 +1,32 @@
+package v1
```

Comment:
The package here should be `paddlepaddle`, instead of `v1`. Please take [this file](https://github.com/google/google-api-go-client/blob/master/sheets/v4/sheets-gen.go) from the Go client library of Google API as a reference.